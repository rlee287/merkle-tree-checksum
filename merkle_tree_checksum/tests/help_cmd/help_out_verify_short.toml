bin.name = "merkle_tree_checksum"
args = "verify-hash -h"
status.code = 1
