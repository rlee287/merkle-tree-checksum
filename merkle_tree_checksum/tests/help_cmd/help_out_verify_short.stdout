merkle_tree_checksum-verify-hash 
Verify Merkle tree hashes

USAGE:
    merkle_tree_checksum verify-hash [OPTIONS] <FILE>

OPTIONS:
        --fail-fast    Bail immediately on hash mismatch
    -h, --help         Prints help information

ARGS:
    <FILE>    File containing the hashes to check
